driver-core/src/main/java/com/datastax/driver/core/TableMetadata.java

```diff
@@ -156,12 +203,13 @@ static TableMetadata build(KeyspaceMetadata ksm, Row row, Map<String, ColumnMeta
                 clusteringOrder.set(i, comparator.reversed.get(i) ? Order.DESC : Order.ASC);
             }
 
+            // FIXME should this hidden column be included in the metadata?
             // We have a value alias if we're dense
-            if (isDense) {
-                String alias = row.isNull(VALUE_ALIAS) ? DEFAULT_VALUE_ALIAS : row.getString(VALUE_ALIAS);
-                DataType type = CassandraTypeParser.parseOne(row.getString(VALIDATOR), protocolVersion, codecRegistry);
-                otherColumns.add(ColumnMetadata.forAlias(tm, alias, type));
-            }
+//            if (isDense) {
```

Comment:
This puzzles me: the code's been there for ages and it creates a column without name that now we need to remove. Will probably raise a ticket to investigate that.